### DISCUSSION TITLE
Error: No download strategy for builder: undefined

### DISCUSSION THREAD

**Discussion by Maister-G** [aclap-dev/video-downloadhelper#71] [Help and Q&A]
### Your question

Al intentar descargar un video de los servidores Gamovideo o Powvideo aparece una notificacion de color rojo mostrando el siguiente Error: No download strategy for builder: undefined (detalles) 
at yT (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:6803)
at async Zr (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:12666)

### Addon & CoApp version and details

Video DownloadHelper
Versión 8.2.0.19 (stable)
Idioma del Navegador: es-ES
Compilación de la plataforma
Compilado el 2024-02-11
Opciones de compilación: browser=chrome, linuxlic=false, noyt=true, target=google

Plataforma Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36

Encontrada Aplicación Auxiliar: VdhCoApp 2.0.10
Ruta de la Aplicación Auxiliar: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by paulrouget** [comment]
Ah, that's a bug. I'll look into it.

**Comment by paulrouget** [comment]
Can you give me the URL of a failing video?

**Reply by Maister-G** [reply]
https://gamovideo.net/2y82z0gn2nla
https://powstream.one/j4wrp0qmaayp

**Reply by paulrouget** [reply]
Ok! Found the bug.

I updated the addon. Tomorrow you'll see version `8.2.0.20` which hopefully fixes your problem.

Thank you!

**Reply by Maister-G** [reply]
Gracias a ti y a tu equipo por vuestro gran trabajo con esta excelente aplicacion la cual llevo usando muchos años.un saludo
